packages/cascara/src/modules/DataEmail/DataEmail.js

```diff
@@ -23,21 +23,22 @@ const DataEmail = ({
   attribute,
   isEditable = true,
   isLabeled = true,
-  label,
   value,
   ...rest
 }) => {
   const { isEditing, formMethods } = useContext(ModuleContext);
 
   const renderEditing = (
-    <label htmlFor={label}>
-      {label && isLabeled && <span className={styles.Label}>{label}</span>}
+    <label htmlFor={rest.label}>
```

Comment:
@lordpixel why did we remove label from the object destructuring?

Reply:
This is a mistake, this file should have not been committed, let me remove it.